### DISCUSSION TITLE
Reference is included in 2-page limit in assignment3?

### DISCUSSION THREAD

**Discussion by junsu-kim97** [seominjoon/kaist-ai605#43] [Q&A] [answered]
Dear professor and TAs,

Thanks for the great course!

I have a question about the 2-page limit in the assignment 3.

Is "Reference" not included in the 2-page limit?

I couldn't find related info in the assignment docs, so I post this question.

Best,

Junsu Kim

**Comment by seominjoon** [comment, marked as answer]
No, reference is not included. I will put it in the assignment as well.
